### ISSUE TITLE
Nest server not found!

### ISSUE THREAD

**Issue by nosratullah** [babsey/nest-desktop-legacy#4]
I got this error:
`Simulation failed. NEST Server not found. Please check the configuration.`
could you please give me a help?
I installed my nest with `conda` command.

**Comment by babsey** [comment]
This error means that NEST Desktop cannot find NEST Server. 
Be sure that NEST Server is running (in other terminal session or as a service).

In app configuration of NEST Desktop you can enter the host and port of NEST Server if it is not serving at localhost:5000.

By the way, NEST Desktop can be installed via pip3 (pip3 install nest-desktop) and it will also install NEST Server. 

To start NEST Server (with environment of NEST Simulator): nest-server start

To start NEST Desktop: nest-desktop start

Good luck.

**Comment by BSVogler** [comment]
Will the nest server integration change once [your PR](https://github.com/nest/nest-simulator/pull/1415) is integrated into the official nest repo?

**Comment by babsey** [comment]
It is off-topic. Please make a new issue.

> Will the nest server integration change once [your PR](https://github.com/nest/nest-simulator/pull/1415) is integrated into the official nest repo?

Yes, I am preparing NEST Desktop to use NEST Server from offical NEST repo.

**Comment by BSVogler** [comment]
Sorry for the off-topic question. I thought that issues with dependencies might resolve themselves when it becomes a part of nest.
After your talk on the nest conference I wanted to try it nest dekstop. I installed both via pip. I start nest-server and nest-desktop. nest server starts on port 500 so I changed it in the settings. Unfortunately, I still get an error message that the nest server is not running.
![Bildschirmfoto 2020-06-30 um 15 46 07](https://user-images.githubusercontent.com/801818/86133612-d2cc3a00-bae8-11ea-9af0-9bc3765f1ab7.png)

**Comment by babsey** [comment]
I assume that NEST Server is not started properly.
 - Can you check which version of NEST Server do you use?
 - Try this command to show outputs 'nest-server start -o' and can you find the source of the error?

**Comment by BSVogler** [comment]
NEST Server 2.4.1

```
Traceback (most recent call last):
  File "/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/nest_server/main.py", line 13, in <module>
    import nest.topology as topo
ModuleNotFoundError: No module named 'nest.topology'
```

I guess it expects nest 2.20? My nest installation is nest 3. I think if the server fails to start it should print something. nosratullah may had a similar issue with the nest installation.

**Comment by babsey** [comment]
Last Friday I released NEST Desktop 2.5.0 and NEST Server 2.5.0. 
This is last release of NEST Server from my side. 
NEST Simulator has implemented NEST Server and can be used in NEST 3.
NEST Desktop 2.5.0 could work with NEST Server from NEST 3.

**Comment by babsey** [comment]
Closed because it has been solved in other repository.
